Makefile

```diff
@@ -921,9 +921,15 @@ else
 LIBHALIDE_SONAME_FLAGS=
 endif
 
+ifeq ($(UNAME), Linux)
+LIBHALIDE_EXPORTS=-Wl,--version-script=$(ROOT_DIR)/src/exported_symbols.linux
+else
+LIBHALIDE_EXPORTS=-Wl,-exported_symbols_list $(ROOT_DIR)/src/exported_symbols.osx
```

Comment:
Should this not be `-Wl,$(ROOT_DIR)/src/exported_symbols.osx`?

Reply:
What's there is what works locally for me. I'll retest this morning to be sure I'm not hallucinating.

Reply:
> What's there is what works locally for me. I'll retest this morning to be sure I'm not hallucinating.

Verified, working properly for me with my local XCode 12.x setup

Reply:
Huh, okay. I guess the driver is smart enough to pass that on automatically.